docs/core/install/linux-package-mixup.md

```diff
@@ -141,10 +149,37 @@ If your distribution provides .NET packages, it's recommended that you use that
         sudo apt-get update && sudo apt-get install -y dotnet-sdk-7.0
         ```
 
+    When .NET is installed to different folders, set the `DOTNET_ROOT` environment variable to the .NET directory required by your apps. For more information, see [.NET SDK and CLI environment variables](../tools/dotnet-environment-variables.md#net-sdk-and-cli-environment-variables).
+
 04. **I've encountered a bug in the Linux distribution version of .NET, I need the latest Microsoft version.**
 
     Use solution 3 to solve this problem.
 
+05. **I need multiple versions of .NET, which aren't available in the Linux distribution repositories.**
```

Comment:
This is not going to work... that's the reason we wrote this doc: when both the distro and Microsoft provide .NET, things break in all sorts of weird ways unless we specifically pick dotnet from one provider :/